soh/soh/Enhancements/game-interactor/GameInteractionEffect.cpp

```diff
@@ -380,10 +282,10 @@ namespace GameInteractionEffect {
         }
     }
     void WeatherRainstorm::_Apply() {
-        GameInteractor::RawAction::SetWeatherStorm(1);
+        GameInteractor::RawAction::SetWeatherStorm(true);
```

Comment:
The reason I always used 1/0 instead of true/false is because otherwise it can't be used in C code down the line. Not sure if there's a way around that considering we most likely need wrapper functions for using these in C anyway, but might be worth thinking about.